src/lib/components/Swap/Settings/MaxSlippageSelect.tsx

```diff
@@ -1,9 +1,10 @@
-import { t, Trans } from '@lingui/macro'
+import { Trans } from '@lingui/macro'
```

Comment:
You've still changed the UI here. You shouldn't be changing the `<Option>` element here except to update it to say `Auto`. Please back most of this out.

In particular, you no longer display the checkmark when the option is selected. This is a UI regression.